lib/cookbook-omnifetch/artifactserver.rb

```diff
@@ -114,10 +121,28 @@ def ==(other)
       other.rel == rel
     end
 
+    # The path where all pristine tarballs from an artifactserver are held.
+    # Tarballs are moved/swapped into this location once they have been staged
+    # in a co-located staging directory.
+    #
+    # @return [Pathname]
     def cache_root
       Pathname.new(CookbookOmnifetch.cache_path).join('.cache', 'artifactserver')
     end
 
+    # The path where tarballs are downloaded to and unzipped.  On certain platforms
+    # *cough* Windows *cough*, you have a better chance of getting an atomic move
```

Comment:
This is true of Windows and Unix, e.g., if you mount /tmp/ as a different disk, then `mv /tmp/foo /elsewhere/foo` isn't atomic.